Core/Workflow/Utility.py

```diff
@@ -1,36 +1,37 @@
-# $Id$
-
 """
     Workflow Utility module contains a number of functions useful for various
     workflow operations
 """
 
 __RCSID__ = "$Id$"
 
-import types, re
+import re
+
 
-def getSubstitute(param,skip_list=[]):
+def getSubstitute(param, skip_list=[]):
```

Comment:
I suppose this is all automated changes, but don't you want to fix it and use sets ?

Reply:
These are automated changes, yes.

Anyway, to answer your question: maybe yes, but not in this PR, which is for a patch. 

This Core/Workflow package is central and extremely delicate, I am reluctant to do any important change there in a patch.

Reply:
OK. Sure, first thing will be to have unit test :-)